### DISCUSSION TITLE
Using MkDocs from Github Actions with Material - Plugins not enabled?

### DISCUSSION THREAD

**Discussion by dcaddick** [squidfunk/mkdocs-material#4870] [Q&A]
Hi, first time posting here in Github Discussions.

I have managed to use MkDocs & Material via Github Actions to publish a website to Github pages.
And I will freely admit that I'm very dangerous and chiefly code with a chainsaw and sledge hammer...

What I'm finding difficult to understand is how the Plugins work?
In my Github workflow I have understood enough to install plugins via pip here:
[MakeDocsAction.yaml](https://github.com/dcaddick/gsd_public/blob/main/.github/workflows/MakeDocsAction.yaml)
But also curious as to whether or not I should be just able to "add" these to the list via the requirements.txt file?
[requirements.txt](https://github.com/dcaddick/gsd_public/blob/main/requirements.txt)

Anyway - getting them installed is one thing, easy enough, the hard part that I'm struggling with is getting them enabled?
See here for the mkdocs.yml in root:
[mydocs.yml](https://github.com/dcaddick/gsd_public/blob/main/mkdocs.yml)

The only ones I can get running are Search, Simple & Macros - am I doing something really dumb? 
Snippet:
```
# Plugins
plugins:
  - search
  - simple
  #- techdocs-core
  #- glightbox
  #- social
  #- awesome-list
  #- table-reader
  #- img2fig
  #- mermaid2
  - macros
  # - minify
```

**Comment by squidfunk** [comment]
The `#` is a YAML comment – this means the plugins are commented out, thus not enabled. Other than that, it's impossible to know what might be wrong without a reproduction, and/or at least an error message what's going sideways. In general, you need to install plugins, enabled them and configure them accordingly to our docs (= natively supported) or according to the plugin developer's docs.

**Reply by dcaddick** [reply]
Thanks, I've commented out the Plugins that keep erroring.
Here is an example of the error "Action"
[Action run - with error](https://github.com/dcaddick/gsd_public/actions/runs/3922014831)

Output:
![image](https://user-images.githubusercontent.com/19640455/212651626-1ed68f4b-6148-42c1-8c12-ce4b4f8a8739.jpeg)

**Reply by dcaddick** [reply]
I've simply not worried about the Plugins errors and got on with creating the content and figured I'd get around to this at some stage.

Using pip I can consistently get the plugin to "install" but getting it enabled has consistently shown an error similar to this with just the name of the plug-in changing.

I figured it was just me doing something dumb?
